--- origsrc/luasvn-0.2.6/src/makefile	2007-10-24 17:11:23.000000000 -0500
+++ src/luasvn-0.2.6/src/makefile	2008-02-13 23:09:28.718750000 -0600
@@ -3,18 +3,18 @@
 APR_UTIL=-I/usr/include/apr-1.0
 
 
-CFLAGS=-Wall -O2 -fpic -D_REENTRANT -D_GNU_SOURCE -D_LARGEFILE64_SOURCE -ggdb \
+CFLAGS=-Wall -O2 -DPIC -D_REENTRANT -D_GNU_SOURCE -D_LARGEFILE64_SOURCE -ggdb \
  -fomit-frame-pointer -pipe -pthread $(SUBVERSION) $(APR) $(APR_UTIL)
 
 # If the Subversion is not in the default path, use the -Wl,-R flag
 #LDFLAGS=-Wl,-R/usr/local/lib -O -shared -fpic  
-LDFLAGS=-O -shared -fpic 
+LDFLAGS=-O -shared -Wl,--enable-auto-image-base
 
 # --- 
 
-LIBS=-lsvn_client-1 
+LIBS=-lsvn_client-1
 
-TARGET=svn.so
+TARGET=svn.dll
 
 OBJS=luasvn.o
 CC=gcc
